pyproject.toml

```diff
@@ -28,14 +28,13 @@ dependencies = [
   "numpy>=1.23",
   "scipy>=1.9",
   "ortools==9.8.3296",
+  "rasterio>=1.2",
+  "h5py>=3.6",
+  "tqdm>=4.60",
```

Comment:
This is a question for this team - if they want progress bars or not. If users don't need to know progress - can remove it. This was inherited from `dolphin` to show progress bars but nothing has been wired up yet.

Reply:
Thanks @piyushrpt for addressing all of my comments.

I think the question of whether we want progress bars or not comes down to whether we think of spurt as more of [a library or an application](https://iscinumpy.dev/post/app-vs-library/). A library should avoid adding dependencies like this, whereas it'd be fine for an application. Dolphin and spurt both kind of straddle the boundary between the two since both have a user-facing command line interface but both are also used as upstream dependencies of higher-level workflows.

I don't have strong feelings about whether or not we include the progress bars (though personally I would forego them). 

But I just don't think we should include this dependency if we don't actually use it. So let's please remove it unless/until we introduce progress bars somewhere.

Reply:
Done